src/assets/scripts/aircraft/AircraftFlightManagementSystem.js

```diff
@@ -72,12 +82,15 @@ export default class AircraftFlightManagementSystem {
             anything: false    // T/F flag for if anything is being "followed"
         };
 
+        // TODO: enumerate the magic numbers
         // set initial
         this.fp.altitude = clamp(1000, options.model.ceiling, 60000);
 
         if (options.aircraft.category === 'arrival') {
+            // TODO: why KDBG?
```

Comment:
I feel like this is confusing.  Perhaps we could update this to be a little clearer like `UNASSIGNED` or something.  I'll add an issue for future consideration #102.